modules/mono/glue/GodotSharp/GodotSharp/Core/Bridge/ScriptManagerBridge.cs

```diff
@@ -298,6 +298,12 @@ static void LookupScriptForClass(Type type)
 
                 _pathTypeBiMap.Add(scriptPathAttr.Path, type);
 
+                if (Engine.IsEditorHint())
```

Comment:
I found another one! This crashes in exported projects because this is reached before `InitializeFromGameProject` returns and managed callbacks get set.

Stack:

```
 	Net7.exe!CSharpLanguage::setup_csharp_script_binding(CSharpScriptBinding & r_script_binding, Object * p_object) Line 1269	C++
 	Net7.exe!CSharpLanguage::get_instance_binding(Object * p_object) Line 1437	C++
 	Net7.exe!godotsharp_internal_unmanaged_get_instance_binding_managed(Object * p_unmanaged) Line 236	C++
 	[Managed to Native Transition]	
 	GodotSharp.dll!Godot.NativeInterop.InteropUtils.UnmanagedGetManaged(System.IntPtr unmanaged) Line 35	C#
 	GodotSharp.dll!Godot.NativeInterop.InteropUtils.EngineGetSingleton(string name) Line 93	C#
 	GodotSharp.dll!Godot.Engine.Singleton.get() Line 122	C#
 	GodotSharp.dll!Godot.Engine.IsEditorHint() Line 526	C#
>	GodotSharp.dll!Godot.Bridge.ScriptManagerBridge.LookupScriptsInAssembly.__LookupScriptForClass|13_0(System.Type type) Line 299	C#
 	GodotSharp.dll!Godot.Bridge.ScriptManagerBridge.LookupScriptsInAssembly(System.Reflection.Assembly assembly) Line 352	C#
 	Net7.dll!GodotPlugins.Game.Main.InitializeFromGameProject(System.IntPtr godotDllHandle, System.IntPtr outManagedCallbacks, System.IntPtr unmanagedCallbacks, int unmanagedCallbacksSize) Line 26	C#
 	[Native to Managed Transition]	
 	Net7.exe!GDMono::initialize() Line 430	C++
 	Net7.exe!CSharpLanguage::init() Line 133	C++
 	Net7.exe!ScriptServer::init_languages() Line 214	C++
 	Net7.exe!Main::setup2(unsigned __int64 p_main_tid_override) Line 2318	C++
 	Net7.exe!Main::setup(const char * execpath, int argc, char * * argv, bool p_second_phase) Line 1848	C++
 	Net7.exe!widechar_main(int argc, wchar_t * * argv) Line 164	C++
 	Net7.exe!_main() Line 205	C++
 	Net7.exe!main(int argc, char * * argv) Line 217	C++
 	[Inline Frame] Net7.exe!invoke_main() Line 102	C++
 	Net7.exe!__scrt_common_main_seh() Line 288	C++
 	kernel32.dll!BaseThreadInitThunk()	Unknown
 	ntdll.dll!RtlUserThreadStart()	Unknown
```

Project is nothing specific, just a single scene with a node with a script that does nothing.

Reply:
To fix this I've added a method to check if the .NET module is initialized (`godotsharp_dotnet_module_is_initialized`).